telegram/bot.py

```diff
@@ -154,6 +154,8 @@ def decorator(self, *args, **kwargs):
             if result is True:
                 return result
 
+            self.create_references(result)
+
```

Comment:
I would tend to agree, I added it for the tests because I used the return value of a sent message to "reply" to it.